.github/ISSUE_TEMPLATE/new_resource.yml

```diff
@@ -11,18 +11,10 @@ body:
       required: true
     attributes:
       label: Approved?
-      description: Has your content (document and headline) been approved by @esquared415? If No, please allow 3-5 business days for review by Erika.
-      multiple: false
+      description: Has your content (document and headline) been approved been approved for publication? If not, please confirm approval in #comms-and-content.
```

Comment:
Can we make this a link to the [`#comms-and-content`](https://cal-itp.slack.com/archives/C01LVK0C4NS) channel? I think this might be interpreted as a comment in the YAML file without at least wrapping in back-ticks

Reply:
Checked that both links are working: <img width="976" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/9b9f12cc-a792-4b2d-bb2a-03cb82314cf0">